### PR TITLE
Heat-Map

### PR DESCRIPTION
**Pull request by martinnajjar12**
### This is a heat-map for the Smashing Magazine:

We've built this page with:
- HTML5
- CSS3: Styling and Positioning.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,477 @@
...
+        </div>
+        <!-- ARTICLES -->
+        <div class="more-articles-div d-flex">
+          <div class="article1">
+            <div class="time b-radius-10"></div>
+            <div class="title b-radius-10"></div>
+            <div class="citing b-radius-10"></div>
+            <div class="paragraph b-radius-10"></div>
+            <div class="second-arrow b-radius-10"></div>
+          </div>
```
Comment (expjazz) [inline index.html]: - [x] This is the main reason the project was not approved. All the articles should be covered by article tags.

image

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

#### Hello Team :raised_hands: 

You did a splendid job on this project. The design looks quite good, and the code is very organized and clean :clap: 

However, there are two minor details that we need to fix in order to move forward:

- [x] The layout looks different. On 1440px it has 2 rows of articles with 3 columns, and then the 3 rows of 3 columns each. In yours is different. As stated here:
https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/pageLayoutScreenSize.png

 _If it shows different in your screen for some reason (in this screen size), kindly send a print screen for the next review._

| Yours | Original |
|:---:|:---:|
|image | image |

:arrow_up: 
Those print screens were took with 1440 px size

- [x] You should wrap all the articles in articles tags (check the inline comments for more)

#### As stated above, the project looks really good.

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

really awesome job! this is one of the best if not the best work I've seen in this project! Good Work 🎉 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by martinnajjar12** [comment]
> # Project Approved ✔️
> really awesome job! this is one of the best if not the best work I've seen in this project! Good Work 🎉
> 
> thank you for making the required changes, your project is now approved!
> 
> ### please don't forget to merge to master, good luck with the next projects!
> Best,
> Aziz Mejri
> _contact me on [twitter](https://twitter.com/iSpaYco) or slack (@Spayco) if you need any help_

Thank You for the nice words and feedback!!

**Pull request merged**